tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -212,30 +212,37 @@ version of the raw dataset will probably go away.
   <!-- ======================== -->
 
   <mksrf_fvegtyp>
-    <!-- Potential Vegetation dataset -->
+    <!-- Potential Vegetation dataset TODO Peter Lawrence? -->
     <entry pft_years="PtVg" >
       <data_filename>lnd/clm2/rawdata/pftcftdynharv.0.25x0.25.LUH2.noanthro.c20230226/mksrf_landuse_ctsm52_noanthroLUH2_1.c20230226.nc</data_filename>
       <mesh_filename>lnd/clm2/mappingdata/grids/UNSTRUCTgrid_0.25x0.25_nomask_cdf5_c200129.nc</mesh_filename>
       <lake_filename>lnd/clm2/rawdata/lake_area/mksurf_lake_0.05x0.05_hist_clm5_hydrolakes_1850.cdf5.c20220325.nc</lake_filename>
       <urban_filename>lnd/clm2/rawdata/mksrf_urban_0.05x0.05_zerourbanpct.cdf5.c181014.nc</urban_filename>
     </entry>
 
+    <entry pft_years="1700">
+      <data_filename>/glade/campaign/cesm/development/lmwg/landuse_source_data/CTSM53RawData/globalctsm53histTRENDY2024Deg025_240728/mksrf_landuse_ctsm53_histTRENDY2024_1700.c240728.nc</data_filename>
```

Comment:
More files to move to inputdata and rimport.

Reply:
All files under /CTSM53RawData are covered by my earlier comment, so I will resolve this one.